static/js/util/ProfileFormFields.js

```diff
@@ -64,13 +66,41 @@ export default class ProfileFormFields extends React.Component {
     }));
   }
 
+  stepBack = () => {
+    this.context.router.push(this.prevUrl);
+  };
+
   saveAndContinue = () => {
-    let lastTab = this.nextUrl === "/dashboard";
-    saveAndContinue.call(this, lastTab).then(() => {
+    saveProfileStep.call(this, this.isLastTab).then(() => {
       this.context.router.push(this.nextUrl);
     });
   };
 
+  progressControls = (cellWidth=10) => {
+    let prevButton, nextButton;
+    if(this.prevUrl) {
+      prevButton = <Button
+        raised
+        className="progress-button previous"
+        onClick={this.stepBack}>
+        <span>Previous</span>
+      </Button>;
+    }
+    if(this.nextUrl) {
+      nextButton = <Button
+        raised
+        colored
+        className="progress-button next"
+        onClick={this.saveAndContinue}>
+        <span>{this.isLastTab ? "I'm Done!" : "Save and Continue"}</span>
+      </Button>;
+    }
+    return <Cell col={cellWidth}>
+        {prevButton}
+        {nextButton}
+      </Cell>;
```

Comment:
💯  understood. thanks for the perspective. i imagine this is something that snags React newbies pretty frequently